gateway/src/commands/mod.rs

```diff
@@ -15,30 +15,66 @@ pub(crate) mod run;
 pub(crate) mod sign;
 pub(crate) mod upgrade;
 
-pub(crate) const ID_ARG_NAME: &str = "id";
-pub(crate) const HOST_ARG_NAME: &str = "host";
-pub(crate) const MIX_PORT_ARG_NAME: &str = "mix-port";
-pub(crate) const CLIENTS_PORT_ARG_NAME: &str = "clients-port";
-pub(crate) const VALIDATOR_APIS_ARG_NAME: &str = "validator-apis";
-#[cfg(not(feature = "coconut"))]
-pub(crate) const VALIDATORS_ARG_NAME: &str = "validators";
-#[cfg(not(feature = "coconut"))]
-pub(crate) const COSMOS_MNEMONIC: &str = "mnemonic";
-#[cfg(not(feature = "coconut"))]
-pub(crate) const ETH_ENDPOINT: &str = "eth_endpoint";
-pub(crate) const ANNOUNCE_HOST_ARG_NAME: &str = "announce-host";
-pub(crate) const DATASTORE_PATH: &str = "datastore";
-pub(crate) const TESTNET_MODE_ARG_NAME: &str = "testnet-mode";
-pub(crate) const WALLET_ADDRESS: &str = "wallet-address";
-
-#[cfg(not(feature = "coconut"))]
+#[cfg(all(feature = "eth", not(feature = "coconut")))]
 const DEFAULT_ETH_ENDPOINT: &str = "https://rinkeby.infura.io/v3/00000000000000000000000000000000";
-#[cfg(not(feature = "coconut"))]
+#[cfg(all(feature = "eth", not(feature = "coconut")))]
 const DEFAULT_VALIDATOR_ENDPOINT: &str = "http://localhost:26657";
 // A dummy mnemonic
-#[cfg(not(feature = "coconut"))]
+#[cfg(all(feature = "eth", not(feature = "coconut")))]
 const DEFAULT_MNEMONIC: &str = "typical regret aware used tennis noise resource crisp defy join donate orient army item immense clean emerge globe gift chronic loan flat enter egg";
```

Comment:
Uhm... I'm slightly confused by why that even exists

Reply:
(i.e. the `DEFAULT_MNEMONIC`)

Reply:
Seems like it was added very recently, so thought I'd leave it in